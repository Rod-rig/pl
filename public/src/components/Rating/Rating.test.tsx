import * as React from "react";
import { create } from "react-test-renderer";
import { Rating } from "./";

describe("Rating", () => {
  const comp = <Rating />;
  const tree = create(comp).toJSON();
  it("should ", () => {
    expect(tree).toMatchSnapshot();
  });
});
