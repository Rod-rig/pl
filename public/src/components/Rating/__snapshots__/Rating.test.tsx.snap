// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rating should  1`] = `null`;
